tests/test_api.py

```diff
@@ -81,7 +81,7 @@ def test_status_format(self, client, endpoint, method):
         ('/family-relationships', ['participant1', 'participant2']),
         ('/phenotypes', ['participant']),
         ('/outcomes', ['participant']),
-        ('/diagnoses', ['participant', 'biospecimen']),
+        ('/diagnoses', ['participant']),
```

Comment:
shouldn't the link to biospecimen also exist?

Reply:
ya it should be `biospecimens`